docs/conf.py

```diff
@@ -57,10 +57,10 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.9'
+version = u'1.0'
 # The full version. Since we adhere to semver, there is no reason to make this
 # more specific.
-release = u'0.9'
+release = u'1.0'
```

Comment:
FYI, it's possible to get conf.py to pull this from pyproject.toml:

https://github.com/ni/measurementlink-python/blob/main/_docs_source/conf.py

This is outside the scope of this PR.

Reply:
ooh nice, I'll play with that after this!